leadiv/directory-poc
# feat: create deploy pipeline

## What's New?
> Creates a deploy pipeline for the pictorial directory.

## Tickets
> #1 

## How To Test?
> 1. Change branch from master to the branch you are testing on. 
> 2. Push the change up on github and watch the deploy job in the actions
> 3. It should push up any of the files in the app folder up to the server.
> 4. Confirm that the files are deployed on http://files.cbclilburn.org
